pepper-apis/src/main/java/org/broadinstitute/ddp/model/dsm/ParticipantStatusTrackingInfo.java

```diff
@@ -12,29 +14,28 @@
 // All epoch timestamps are expressed in seconds
 public class ParticipantStatusTrackingInfo {
 
+    private static final Logger LOG = LoggerFactory.getLogger(ParticipantStatus.class);
+
     public enum RecordStatus {
         INELIGIBLE, PENDING, SENT, RECEIVED, DELIVERED, UNKNOWN
     }
 
-    private static final Logger LOG = LoggerFactory.getLogger(ParticipantStatus.class);
-
-    @SerializedName("medicalRecords")
+    @SerializedName("medicalRecord")
     private MedicalRecord medicalRecord;
-    @SerializedName("tissue")
+    @SerializedName("tissueRecord")
```

Comment:
This is an agreed-upon API change to rename the property names to `medicalRecord` and `tissueRecord`. We also have these two names in the OpenAPI docs, so making the code to match.

Reply:
Thanks for cleaning these up!